webservices/resources/sched_e.py

```diff
@@ -84,3 +93,45 @@ def filter_election(self, query, kwargs):
         if kwargs['district']:
             query = query.filter(models.CandidateHistory.district == kwargs['district'])
         return query
+
+
+@doc(
+    tags=['independent expenditures'],
+    description=docs.SCHEDULE_E,
+)
+class ScheduleEEfileView(views.ApiResource):
+    model = models.ScheduleEEfile
+    schema = schemas.ItemizedScheduleEfilingsSchema
+    page_schema = schemas.ScheduleEEfilePageSchema
+
+    filter_multi_fields = [
+        #('cycle', sa.func.get_cycle(models.ScheduleE.report_year)),
+        ('image_number', models.ScheduleEEfile.image_number),
+        ('committee_id', models.ScheduleEEfile.committee_id),
+        ('candidate_id', models.ScheduleEEfile.candidate_id),
+        ('support_oppose_indicator', models.ScheduleEEfile.support_oppose_indicator),
+        ('filing_form', models.ScheduleEEfile.form_type),
+        ('is_notice', models.ScheduleEEfile.is_notice),
+    ]
+
+    @property
+    def args(self):
+        return utils.extend(
+            args.paging,
+            #args.itemized,
```

Comment:
If these are indeed separate from the other itemized models (see note above in the `BaseRawItemized` class), do we need to consider creating separate `args` for them?